uk.co.compucorp.civicrm.hrleaveandabsences/xml/Menu/import.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<menu>
+  <item>
+    <path>civicrm/admin/leaveandabsences/import</path>
+    <page_callback>CRM_HRLeaveAndAbsences_Import_Controller</page_callback>
+    <title>Import Leave/Absence Requests</title>
+    <access_arguments>administer leave and absences</access_arguments>
+  </item>
+</menu>
```

Comment:
I noticed that all the Importers in Core and in hrjobcontract and hrjobroles extension have the xml file for the import separately. I only followed the format. I already added the link [HERE](https://github.com/civicrm/civihr/pull/1983/files#diff-62db90d66196e016278482a8f7c81df3R493) but will check with Mike